Assets/MixedRealityToolkit.Services/InputSystem/GazeProvider.cs

```diff
@@ -293,76 +252,69 @@ public void RaisePointerUp(MixedRealityInputAction mixedRealityInputAction, Hand
                 isDown = false;
                 currentHandedness = Handedness.None;
                 currentInputSource = null;
-                gazeProvider.InputSystem?.RaisePointerClicked(this, mixedRealityInputAction, 0, handedness, inputSource);
-                gazeProvider.InputSystem?.RaisePointerUp(this, mixedRealityInputAction, handedness, inputSource);
+                InputSystem.RaisePointerClicked(this, mixedRealityInputAction, 0, handedness, inputSource);
```

Comment:
Separate thought: would be nice to have a way to enforce that click is always before up, or vice versa.